### PR TITLE
Fix aria-haspopup usage in the admin menu.

### PR DESCRIPTION
**Pull request by afercia**
Trac ticket: https://core.trac.wordpress.org/ticket/43095 

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: src/js/_enqueues/admin/common.js [hunk 1]
```diff
@@ -1837,6 +1839,34 @@ $( function() {
...
+			if ( action === 'add' ) {
+				elements.each( function() {
+					$( this ).attr( 'aria-haspopup', 'menu' );
+				} );
+
+				return;
+			}
+
+			elements.each( function() {
+				$( this ).removeAttr( 'aria-haspopup' ):
```
Comment (joedolson) [inline src/js/_enqueues/admin/common.js]: Switch from colon to semicolon

**Review by joedolson — CHANGES_REQUESTED** [review]

File: src/js/_enqueues/admin/common.js (hunk 1 above)
Comment (afercia) [reply src/js/_enqueues/admin/common.js]: Fixed. My bad vision 😞

**Review by joedolson — APPROVED** [review]
Looks good!

**Comment by joedolson** [comment]
In [r58449]

**Pull request closed without merging**